idem_azurerm/exec/azurerm/compute/image.py

```diff
@@ -31,10 +33,10 @@
       * ``AZURE_GERMAN_CLOUD``
 
 """
-
 # Python libs
 from __future__ import absolute_import
 import logging
+import six
```

Comment:
we don't need six. we're only supporting py3. change that string_types reference to a py3 string.